test/synth/30_viper_unsat.c

```diff
@@ -0,0 +1,79 @@
+// RUN: %sea smt %s --step=small --inline -o %t.sm.inline.smt2
+// RUN: %z3 %t.sm.inline.smt2 fp.spacer.order_children=2 2>&1 | OutputCheck %s
+//
+// RUN: %sea smt %s --step=large --inline -o %t.lg.inline.smt2
+// RUN: %z3 %t.lg.inline.smt2 fp.spacer.order_children=2 2>&1 | OutputCheck %s
+//
+// CHECK: ^unsat$
+
+#define SEA_SYNTH
+#include "seahorn/seahorn.h"
```

Comment:
add header file `seahorn/seasynth.h` with the content
```
#pragma once
#ifdef sassert
SOME ERROR MESSAGE HERE
#endif
#define SEA_SYNT
#include "seahorn/seahorn.h
```

and then use it instead of defining `SEA_SYNTH` each time

Reply:
This header is added in the first commit.